airflow/providers/google/cloud/transfers/gcs_to_sftp.py

```diff
@@ -52,6 +52,9 @@ class GCSToSFTPOperator(BaseOperator):
     :param destination_path: The sftp remote path. This is the specified directory path for
         uploading to the SFTP server.
     :type destination_path: str
+    :param ignore_gcs_path_in_destination: (Optional) When set to False the path of the file
```

Comment:
```suggestion
    :param use_basename: (Optional) When set to False the path of the file
```

Reply:
or keep_directory_structure

Reply:
I renamed it to `keep_directory_structure`. Thanks @mik-laj !